docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+# Percona Monitoring and Management 2.33.0
+
+| **Release date:** | Dec 13, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+
+### High Availability in PMM
+
+
+We are excited to share our roadmap for High Availability (HA) in PMM, a new solution that our users have been anticipating. HA will provide your PMM near-zero downtime and wil ensure monitoring without data loss during a failover.
+
+
+To ensure a smooth rollout, HA in PMM will be introduced in phases, in line with your requirements and feedback, with PMM 2.33.0 being the starting point. 
+
+!!! caution alert alert-warning "Important"
+    PMM HA will be rolled out in three stages. Stage one is included with PMM 2.33.0 and stages two and three will be rolled out next year.
+
+The following features/use cases will be covered as part of the various stages of HA in PMM:
+
+
+|**Stages of HA in PMM**              | **Solutions Provided**          |
+|-------------------------------------|---------------------------------|
+|**Stage one**           |As an initial step towards preventing data loss we have developed the following:<br><br>**Prevent data loss with VM integration for short outages**<br>If PMM Server goes down, metrics will be buffered on the monitored instance until PMM Server comes back online, which is when metrics will get pushed to PMM Server. Note that this only applies to PMM Clients which run in the so-called `push` mode. The same holds true when a brief network outage occurs on the client side.|
+|**Stage two**           |As part of PMM HA stage two in HA we plan to implement the following:<br><br>**HA data sources**<br>As part of stage two we will let the users use external data sources therby decreasing dependency on the file system.
```

Comment:
Can we add 2023 tag for Stage 2 and Stage 3?

Reply:
Sure